### ISSUE TITLE
2 new hearts on death

### ISSUE THREAD

**Issue by TheNimbleNinja** [Cazsius/Spice-of-Life-Carrot-Edition#95]
im having an issue were when the player dies they gain 2 new hearts and i dont have scaling health in my modpack i do have silents other mods silent gems and other addons666 this is happening in my 1.15 modpack

**Comment by juliand665** [comment]
Does it still happen without silent's other mods (including silentlib)? Try to figure out exactly which other mods need to be present to cause the issue, ideally reducing it down to just 2 or 3 mods.

**Comment by TheNimbleNinja** [comment]
i removed sol carrot and no more hearts on death

**Comment by juliand665** [comment]
Yes, that demonstrates our mod is involved, but I bet it doesn't happen with _just_ solcarrot (or does it?). Try to narrow down which other mods bring out this issue in combination with it.

**Comment by TheUntamed** [comment]
I had the same problem and could narrow it down to the _Cyclic_ mod. This also only happens with Cyclic version 0.4.0 and 0.4.1.
pre 0.4 seems to be fine and 0.4.2 also works for me.

**Comment by juliand665** [comment]
Oh yeah, the release notes for cyclic 0.4.2 mention "Heart containers compatibility fix.". @TheNimbleNinja did your problem occur with cyclic and does 0.4.2 fix it?
